packages/nimble-components/src/table-column/enum-text/cell-view/index.ts

```diff
@@ -0,0 +1,61 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
+import { styles } from '../../text-base/cell-view/styles';
+import { template } from '../../text-base/cell-view/template';
+import type { TableColumnEnumCellRecord } from '../../enum-base';
+import type { MappingConfigText } from '../../../mapping/text';
+import { TableColumnTextCellViewBase } from '../../text-base/cell-view';
+import type { TableColumnEnumTextColumnConfig } from '..';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-table-column-enum-text-cell-view': TableColumnEnumTextCellView;
+    }
+}
+
+/**
+ * A cell view for displaying mapped text
+ */
+export class TableColumnEnumTextCellView extends TableColumnTextCellViewBase<
+TableColumnEnumCellRecord,
+TableColumnEnumTextColumnConfig
+> {
+    public override get text(): string {
+        return this.getMappingToRender()!.label;
```

Comment:
```suggestion
        return this.getMappingToRender()?.label;
```
Can't `getMappingToRender()` technically return `null` still (e.g. no `defaultMapping` is available)? If so, I would think we would want a test for that scenario (and change the `!` to a `?`).